core/src/main/java/org/apache/iceberg/CachingCatalog.java

```diff
@@ -47,10 +44,6 @@
  */
 public class CachingCatalog implements Catalog {
 
-  private static final Logger LOG = LoggerFactory.getLogger(CachingCatalog.class);
-  private static final RemovalListener<TableIdentifier, Table> identLoggingRemovalListener =
```

Comment:
Should we be logging this in the new removal listener?

Reply:
I'll put it back again

Reply:
done :D